### DISCUSSION TITLE
Displaying data from Blackbox Exporter

### DISCUSSION THREAD

**Discussion by fcaponi** [flant/grafana-statusmap#174] [Q&A]
Hi,
first of all, many thanks for this great plugin.

i'm wondering if you could include an example showing probe status from the prometheus probe blackbox-exporter.

I'm trying the dashboard found in this project https://github.com/maxandersen/internet-monitoring but unfortunately i get an empty panel and i'm struggling trying to understand whats is wrong.

Many thanks in advance
Franco

**Comment by diafour** [comment]
Hello!

docker-compose will start the latest Grafana, which is 7.4: https://github.com/maxandersen/internet-monitoring/blob/master/docker-compose.yml#L31. There is a bug in the plugin and it is not compatible with 7.4 right now (see #169). Replace that line with `    image: grafana/grafana:7.3` and it should work.

**Reply by fcaponi** [reply]
thanks for the suggestion. With that workaround i get the status panel working.

Unfortunately with the old version I'm getting other problems on other dashboards.

Any plan to support also 7.4?

**Reply by shurup** [reply]
> Any plan to support also 7.4?

Yes, we're currently working on it as it is a highly requested issue (#169). The release should be ready soon.
